pages/pipelines/build_timeouts.md

```diff
@@ -0,0 +1,38 @@
+# Build timeouts
```

Comment:
Should this actually be called "Job timeouts?" When going through the content, it seems to be about timeouts on jobs rather than builds.

Reply:
I found myself asking the same question with you @mbelton-buildkite . It has always been called build timeouts so I assumed that was a better name. 

I can also see why they call it build timeouts also actually as I write this. The end goal is for the build to not be running and reach a terminal state that is finished/cancelled/passed. So although the timeouts are in job level they eventually let the builds get to a terminal state. 

I also worry if job timeouts will confuse customers because of job/step confusion + their end goal of achieving build timeouts. I would rather keep the title but have something at the beginning explaining why like what you have done :)

Reply:
👍🏻 Cool, glad I wasn't alone in that thought haha. Happy to leave it as-is.